packages/origin/src/blockchain-facade/Certificate.ts

```diff
@@ -131,14 +133,21 @@ export const getAllCertificateEvents = async (
 
 export class Entity extends BlockchainDataModelEntity.Entity implements ICertificate {
     public assetId: number;
+
     public generationStartTime: number;
+
     public generationEndTime: number;
 
     public owner: string;
+
     public energy: number;
+
     public status: Status;
+
     public creationTime: number;
+
     public parentId: number;
+
     public children: string[];
```

Comment:
Are these whitespace add manually or we somehow have different prettier configs?

Reply:
Yeah this is added automatically, not a fan of this either. We should look into removing it IMO.